posts/2022-08-04-jakarta-core-profile-22009-beta.adoc

```diff
@@ -109,7 +102,7 @@ If you want to use a third-party Jakarta JSON Processing 2.1 or Binding 3.0 impl
 <bell libraryRef="myJsonProcessingProvider"/>
 ----
 
-NOTE: When you enable `jsonb-3.0` or `jsonbContainer-3.0` the `jsonp-2.1` feature is automatically enabled. However, enabling the `jsonpContainer-2.1` feature supersedes the `jsonp-2.1` feature and allows you to provide a third-party implementation.
+Enabling either the JSON-B 3.0 feature (`jsonb-3.0`) or the JSON-B 3.0 Container (`jsonbContainer-3.0`) feature automatically enables the JSON-P 2.1 feature (`jsonp-2.1`), unless you have enabled the JSON-P Container feature (`jsonpContainer-2.1`) in order to provide a third-party implementation of JSON-P.
```

Comment:
Personally I think the `NOTE:` here that the dev used was appropriate, as it's not about the functionality that the rest of the section describes.  Instead, it's a detail/aside.

Reply:
Yeah...I've changed it to "Note that..." (just because that's what I tend to use in that context).